src/clusterfuzz/_internal/bot/tasks/utasks/symbolize_task.py

```diff
@@ -184,7 +196,7 @@ def utask_main(uworker_input):
                                  old_crash_stacktrace, sym_crash_state))
 
   symbolize_task_output = uworker_msg_pb2.SymbolizeTaskOutput(
-      crash_type=sym_crash_type,
+      crash_type=testcase.crash_type,
```

Comment:
Hmm..shouldn't this be state.crash_type in some cases?

Reply:
`state.crash_type` is always equal to `testcase.crash_type`, see line 151.

Reply:
I'm confused. Doesn't 151 imply they are not always the same?

Reply:
It's confusing!

On line 151 what happens is that we *skip* the new big-redzone crash if its type does not match the original crash type from `testcase`. In other words, we only use an ASAN big-redzone crash if its type is the same as `testcase.crash_type`.

Overall, the only parameters that are allowed to change are the address and crash state. I don't really understand what the crash state entails, so I can't tell if the crash type being fixed means the comment above on line 121 is a lie or not:

```
# A UAF might actually turn out to be OOB read/write with a bigger redzone.
```

Reply:
I removed it because I think it was useless. I tried to lay out my reasoning in the PR description, but I had a hard time making it really clear. Basically, I think we were assigning the same value twice to `sym_crash_type`.

Reply:
Oh sorry for not reading the description well enough then.

Reply:
No worries! Thanks for checking!